### PR TITLE
docs: updates minimum required graalvm

### PR DESCRIPTION
**Pull request by machi1990**
See [NativeImagePhase#isThisGraalVMRCObsolete](https://github.com/quarkusio/quarkus/blob/5d4db9dc206910c88c48891df6baa037269b9148/core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java#L527)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ There is documentation published at <https://quarkus.io> (docs' [sources are her
 
 * [Maven 3.5.3+](https://maven.apache.org/install.html)
 * [Java - OpenJDK 1.8+](https://adoptopenjdk.net/)
-* [GraalVM rc14+](https://www.graalvm.org/)
+* [GraalVM rc15+](https://www.graalvm.org/)
```
Comment (cescoffier) [inline README.md]: It's going to be pretty hard to keep this value up to date.
The recommended version is documentation on the web site. Maybe adding a link to the web site would be ok. 
@gsmet WDYT?

Comment (starksm64) [reply README.md]: Why not have a general requirements page that goes over all of the dependency requirements including the os level ones as I have to track this down whenever I setup a new system. Having that as an initial link in every guide would ensure consistency.

Comment (machi1990) [reply README.md]: > It's going to be pretty hard to keep this value up to date.

Totally agree with you. 

> Why not have a general requirements page that goes over all of the dependency requirements including the os level ones as I have to track this down whenever I setup a new system. Having that as an initial link in every guide would ensure consistency.

+1

Comment (gsmet) [reply README.md]: Agreed.

Comment (machi1990) [reply README.md]: > The recommended version is documentation on the web site. Maybe adding a link to the web site would be ok.

Is this link that we are referring? https://quarkus.io/guides/building-native-image-guide

**Comment by rsvoboda** [comment]
fyi: PR https://github.com/quarkusio/quarkus-quickstarts/pull/181 pushes line 34 to GraalVM rc16+

**Comment by gsmet** [comment]
As discussed, I pointed the users to the website where the information is up to date and where you also have additional information on how to set it up.

**Pull request closed without merging**